mon-pix/app/templates/profilv2.hbs

```diff
@@ -21,7 +21,9 @@
       </div>
     </div>
 
-    {{#link-to "compte" class="link link--grey rounded-panel__link"}}Accès à l'ancien profil{{/link-to}}
+    <div class="profilv2__v1link">
+      {{#link-to "compte" class="link link--grey rounded-panel__link"}}Accès à l'ancien profil{{/link-to}}
+    </div>
```

Comment:
J'ai du mal à comprendre les raisons derrière cette manière de faire. Pourquoi un nom aussi spécifique que `profilv2__v1link` ? Mais surtout pourquoi ne pas mettre cette `class` directement sur le `link-to` ?

Reply:
Il faut wrapper le `link-to` afin de pouvoir l'aligner à droite, je suis preneur d'une meilleure solution. Enfin le nom de `profilv2__v1link` peut être changé.

Reply:
En fait après réflexion ça me dérange pas que ce soit très spécifique, ça sera plus simple à supprimer quand on virera v1 comme ça.
Pas de meilleure idée pour le wrap :/

Reply:
Non le lien s'affiche mal :\